test_snakes_cafe.py

```diff
@@ -25,42 +31,75 @@ def test_add_item_adds_multiple_entries():
 
 
 def test_remove_item_test_proper_input():
-    # test remove is recieving expected input
+    """
+    test remove is recieving expected input
+    """
     sc.menu = sc.default_menu
     sc.current.add_item('Coffee', 1)
-    sc.current.receipt == {'subtotal': 1.59, 'Coffee': 1}
+    assert sc.current.receipt == {'Coffee': 5, 'Tea': 1, 'subtotal': 7.95}
 
 
 def test_remove_item_test_remove_single_item():
-    # test remove is recieving expected input
+    """
+    test remove is recieving expected input
+    """
     sc.menu = sc.default_menu
-    sc.current.add_item('Coffee', 2)
+    # sc.current.add_item('Coffee', 1)
     sc.current.remove_item('Coffee', 1)
-    sc.current.receipt == {'subtotal': 1.59, 'Coffee': 1}
+    assert sc.current.receipt == {'Coffee': 4, 'Tea': 1, 'subtotal': 6.36}
 
 
 def test_remove_item_test_remove_multiple_item():
-    # test remove is recieving expected input
+    """
+    test remove is recieving expected input
+    """
     sc.menu = sc.default_menu
     sc.current.add_item('Fries', 3)
     sc.current.add_item('Steak', 1)
     sc.current.remove_item('Fries', 2)
     sc.current.receipt == {'subtotal': 3.28, 'Fries': 1, 'Steak': 1}
 
 
+def test_uuid_created():
+    """
+    Test is output is a string
+    """
+    assert type(sc.current.id) == str
+
+
+def test__len__():
+    """
+    Returns length of reciept
+    """
+    sc.menu = sc.default_menu
+    sc.current.add_item('Steak', 1)
+    assert sc.current.__len__() == 4
+
 
+def test__repr__():
+    """
+    Returns repr
+    """
+    assert type(sc.current.__repr__()) == str
```

Comment:
You should assert more than the type of the output also this and len can simply be str/len(sc.current)